community/index.md

```diff
@@ -149,9 +149,9 @@ the community.
 - stripes-loader: Jason Skomorowski (lead)
 - stripes-sample-platform: Jason Skomorowski (lead)
 - okapi-stripes: Wolfram Schneider (lead)
-- ui-users: Mike Taylor, Niels Erik Nielsen, Jason Skomorowski, John Coburn, Matt Connolly, Jeremy Huff
-- ui-items: Mike Taylor
-- ui-okapi-console: Mike Taylor, Niels Erik Nielsen
+- UI-users: Mike Taylor, Niels Erik Nielsen, Jason Skomorowski, John Coburn, Matt Connolly, Jeremy Huff
+- UI-items: Mike Taylor
+- UI-okapi-console: Mike Taylor, Niels Erik Nielsen
 - cql2pgjson-java: Julian Ladisch (lead)
```

Comment:
Please do not change the names of modules. They are explicitly named with lower-case.
See https://dev.folio.org/source-code/map/#ui